src/current/v23.1/log-sql-statistics-to-datadog.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Log SQL statistics to Datadog
+summary: Examples of common logging use cases and possible CockroachDB logging sink configurations.
+toc: true
+docs_area: manage
+---
+
+{% include_cached new-in.html version="v23.1.4" %} This tutorial describes how to configure logging of [`sampled_query` events](eventlog.html#sampled_query) to [Datadog](https://www.datadoghq.com/) for finer granularity and long-term retention of SQL statistics. The `sampled_query` events contain common SQL event and execution details for sessions, transactions, and statements.
+
+CockroachDB supports a built-in integration with Datadog which sends query events as logs via the [Datadog HTTP API](https://docs.datadoghq.com/api/latest/logs/). This integration is the recommended path to achieve high throughput data ingestion, which will in turn provide more query events for greater workload observability.
+
+## Step 1. Create a Datadog API key
+
+1. In Datadog, navigate to [**Organization Settings > API keys**](https://app.datadoghq.com/organization-settings/api-keys).
+1. Follow the steps in the Datadog documentation on how to [add an API key](https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token).
+1. Copy the newly created API key to be used in Step 2.
+
+## Step 2. Configure an HTTP network collector for Datadog
+
+Configure an [HTTP network collector] (configure-logs.html#output-to-http-network-collectors) by creating or modifying the [`log.yaml` file](configure-logs.html#yaml-payload).
+
+{{site.data.alerts.callout_danger}}
+Given the [volume of `sampled_query` events](#step-3-configure-cockroachdb-to-emit-query-events), do not write `sampled_query` events to disk, or [`file-groups`](configure-logs.html#output-to-files). Writing a high volume of `sampled_query` events to a file group will unnecessarily consume cluster resources and impact workload performance. 
+
+To avoid writing to disk and disable file creation:
```

Comment:
I think we should reconsider this portion of the instructions. This directs customers to turn off **all** non-Telemetry logs which would be pretty dangerous for a production cluster and prevent any debugging from proceeding.

I think a better option is to provide a config that **only disables TELEMETRY** file output which is something like the following. @kevin-v-ngo can provide a suggestion here as well from the customer we built this integration for.

```
sinks:
  http-servers:
    datadog:
      channels: [TELEMETRY]
      address: https://http-intake.logs.datadoghq.com/api/v2/logs
      format: json
      method: POST
      compression: gzip
      headers: {DD-API-KEY: "{DATADOG API KEY}"} # replace with actual API key
      buffering:
        format: json-array
        max-staleness: 5s
        flush-trigger-size: 2.5MiB # override default value
        max-buffer-size: 50MiB
  file-groups:
    telemetry:  # override telemetry file defaults; leave the rest alone
      channels: []
```